src/JustSaying/AwsTools/MessageHandling/IMessageDispatcher.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS.Model;
+
+namespace JustSaying.AwsTools.MessageHandling
+{
+    public interface IMessageDispatcher
+    {
+        Task DispatchMessage(Message message, CancellationToken cancellationToken);
```

Comment:
This probably needs renaming - I'm guessing this is the existing "start a task and then rather than await it look at it and do stuff" thing, but as a interface by itself that isn't clear if you were implementing it in isolation.